guides/common/modules/proc_reset-custom-ssl-certificate-to-default-self-signed-certificate-on-project.adoc

```diff
@@ -0,0 +1,27 @@
+[id="reset-custom-ssl-certificate-to-default-self-signed-certificate-on-{project-context}_{context}"]
+= Reset {customssl} certificate to default self-signed certificate on {ProjectServer}
+
+.Procedure
+* Reset the {customssl} certificate to default self-signed certificate:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --certs-reset
+----
+
+.Verification
+ifndef::orcharhino[]
+Ensure the following parameters in `/etc/foreman-installer/scenarios.d/{project-context}-answers.yaml` have no values:
+endif::[]
+ifdef::orcharhino[]
+Ensure the following parameters in `/etc/foreman-installer/scenarios.d/katello-answers.yaml` have no values:
```

Comment:
IMHO that's correct. I'll add it on my TODO list to make this easier by creating an attribute.